R/exercise_0.Rmd

```diff
@@ -45,15 +46,41 @@ In this case, we want to modify the general pattern with:
 - `[Resource] = Patient`
 - [`_id` parameter](https://www.hl7.org/fhir/search.html#id) = `smart-1032702`
 
+```{r}
+fhir_server <- "https://api.logicahealth.org/researchonfhir/open/"
+cs <- fhir_capability_statement(fhir_server)
+##?cs$Resources$typestr_c("FHIR Resources Supported: ", cs$Resources$type %>% length, "\n") %>% cat
+```
+
 ```{r}
 fhir_server <- "https://api.logicahealth.org/researchonfhir/open/"
```

Comment:
```suggestion
# Note: `fhir_server` set up above to `https://api.logicahealth.org/researchonfhir/open/`
```

I would consider not repeating the `fhir_server` assignment after it's initially set. I think I understand why you might want to repeat it as it makes what's going on clearer, it's not a good pattern for participants to follow in their own work (in case they need to change the server URL).

Perhaps changing to a comment here and below would be a good middle ground?